.buildkite/pipeline.yaml

```diff
@@ -1,38 +1,14 @@
 steps:
-- label: "fast"
+- label: "Build, lint, test"
   commands:
   - "cargo fmt -- --check"
   - "cargo test --all"
   agents:
     production: "true"
     platform: "linux"
   env:
-    DOCKER_IMAGE: "gcr.io/opensourcecoin/radicle-link-build@sha256:fc5eded79ac8af38482e4fb54ec6dfb16e04a3fff2c3a061a60602b827b9f54e"
-    DOCKER_FILE: .buildkite/docker/rust/Dockerfile
-
-- label: "Latest Stable Rust"
-  commands:
-  - "cargo test --all"
-  agents:
-    production: "true"
-    platform: "linux"
-  env:
-    DOCKER_FILE: .buildkite/docker/rust/Dockerfile
-  soft_fail: true
-  concurrency_group: "radicle-link/slow"
-  concurrency: 1
-
-- label: "Nightly Rust"
-  commands:
-  - "cargo test --all"
-  agents:
-    production: "true"
-    platform: "linux"
-  env:
+    #DOCKER_IMAGE: "gcr.io/opensourcecoin/radicle-link-build@sha256:"
```

Comment:
Intentional?

Reply:
Yah, waiting for that sha to be mined

Reply:
Ya, saw the next commit as soon as I commented :sweat_smile: